tests/native_loader/src/main/native/com/example/math/math.c

```diff
@@ -16,6 +16,7 @@
 
 #include "com_example_math_NativeMath.h"
 
+// 'JNIEnv*' and 'jclass' args are implicit and must be included by default.
```

Comment:
How about this variant? I don't think it's every correct to leave out these parameters, so `by default` seems out of place.

```suggestion
// 'JNIEnv*' and 'jclass' params are implicit and must be included.
```

Reply:
Agreed -- added.